app/dao/fact_notification_status_dao.py

```diff
@@ -239,7 +239,7 @@ def fetch_notification_status_for_service_for_day(bst_day, service_id):
 
 
 def fetch_notification_status_for_service_for_today_and_7_previous_days(service_id, by_template=False, limit_days=7):
-    start_date = utc_midnight_n_days_ago(limit_days - 1)
+    start_date = utc_midnight_n_days_ago(limit_days)
```

Comment:
I was hesitant to remove the `- 1` here as we added it in [this PR](https://github.com/cds-snc/notification-api/pull/1976/files#diff-4c1d9f51a839bbba6edbea4f69046a7410906281bc359526415b8fbd898fd673R242) for an edge case.